extension/src/background/messageListener/__tests__/popupMessageListener.test.js

```diff
@@ -9,11 +9,24 @@ import {
 } from "background/ducks/session";
 import { decodeString } from "helpers/urls";
 
+const PUB_KEY = "GBOORGNN6F35F3BFI4SF5ZR4Q7VHALNPGRG3MGA6WMOW4BKFOFMNI45O";
+const PRIV_KEY = "SAUIIOB4EB6MZ25RKTKQ6DBXBDKKFQVMPLS2Q5LDH4GAMT7SAQPQMNCB";
+
 console.error = jest.fn((e) => console.log(e));
 
-describe("regular account flow", () => {
-  beforeAll(() => {
-    localStorage.clear();
+async function getKeyId(pubKey, privKey) {
+  const str = pubKey + privKey;
+  const enc = new TextEncoder();
+  const hash = await crypto.subtle.digest("SHA-1", enc.encode(str));
+  return Array.from(new Uint8Array(hash))
+    .map((v) => v.toString(16).padStart(2, "0"))
+    .join("");
+}
+
+describe.skip("regular account flow", () => {
```

Comment:
@piyalbasu I'm hoping you can take a look at this test suite with me, it's the last one I haven't been able to resolve. Basically the KeyManger errors with `Some keys were already stored in the keystore` but I'm not sure of why. We clear local storage in between tests and if I log out the keyManager keys just before it fails I see an empty list 🤔

Reply:
Ah yeah, we can look at that. let's create a new ticket for dealing with any broken tests. We can comment out failing tests just to get this merged